src/main/java/edu/greenblitz/robotName/subsystems/Lifter/NeoLifter.java

```diff
@@ -0,0 +1,59 @@
+package edu.greenblitz.robotName.subsystems.Lifter;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.greenblitz.robotName.utils.DigitalInputMap;
+import edu.greenblitz.robotName.utils.motors.GBSparkMax;
+
+public class NeoLifter implements ILifter {
+    private GBSparkMax motor;
+    private LifterInputsAutoLogged inputs;
+
+    public NeoLifter() {
+        inputs = new LifterInputsAutoLogged();
+        motor = new GBSparkMax(LifterConstants.MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+    }
+
+    @Override
+    public void setPower(double power) {
+        motor.set(power);
+    }
+
+    @Override
+    public void setVoltage(double voltage) {
+        motor.setVoltage(voltage);
+    }
+
+    @Override
+    public void resetEncoder(double position) {
+        motor.getEncoder().setPosition(position);
+    }
```

Comment:
i think its better if you save a reset positon and reset to it (=0)

Reply:
![hatool](https://github.com/GreenBlitz/CRRRRESCENDO/assets/151079574/83d04dbf-38e3-4123-8140-efd92f98abc3)